test/components/cards/lCard.spec.ts

```diff
@@ -20,20 +20,52 @@ describe('LCard component', () => {
   })
 
   it('component lCard exists', () => {
-    expect(card.exists()).toBeTruthy()
+    expect(card.exists()).toBe(true)
   })
 
   it('check props value default forceShowSlot', () => {
-    expect(card.props().forceShowSlot).toBeFalsy()
+    expect(card.props().forceShowSlot).toBe(false)
+  })
+
+  it('check props value default forceShowSlot', () => {
+    expect(card.props().forceShowSlot).toBe(false)
   })
 
   it('check computed show Slot default falsy', () => {
-    expect(card.vm.showSlot).toBeFalsy()
+    expect(card.vm.showSlot).toBe(false)
   })
 
   it('check computed show Slot default truthy', async () => {
     card.setProps({forceShowSlot : true})
     await card.vm.$nextTick()
-    expect(card.vm.showSlot).toBeTruthy()
+    expect(card.vm.showSlot).toBe(true)
+  })
+
+  it('check doesnt show ".LCard--shadow" when props true', async () => {
```

Comment:
O nome do teste está confuso, poderia ser "check if adds LCard--shadow when hasShadow is true"